src/NuGetPackageVerifier/PackageIssueFactory.cs

```diff
@@ -95,6 +95,11 @@ public static PackageVerifierIssue AssemblyNotStrongNameSigned(string assemblyPa
             return new PackageVerifierIssue("SIGN_STRONGNAME", assemblyPath, string.Format("The managed assembly '{0}' in this package is either not signed or is delay signed. HRESULT=0x{1:X}", assemblyPath, hResult), MyPackageIssueLevel.Error);
         }
 
+        public static PackageVerifierIssue AssemblyHasWrongPublicKeyToken(string assemblyPath)
+        {
+            return new PackageVerifierIssue("WRONG_PUBLICKEYTOKEN", assemblyPath, string.Format("The managed assembly '{0}' in this package does not have the correct public key token.", assemblyPath), MyPackageIssueLevel.Error);
```

Comment:
Can the message show the public key token (i.e. `this assembly doesn't use... the b70f54315478 key`) or whatever?